projects/client/RabbitMQ.Client/src/client/impl/Command.cs

```diff
@@ -68,18 +66,22 @@ static Command()
 
         public Command() : this(null, null, null)
         {
+             m_body= new MemoryStream();
         }
 
         public Command(MethodBase method) : this(method, null, null)
-        {
+        {
+            m_body = new MemoryStream();
         }
 
         public Command(MethodBase method, ContentHeaderBase header, byte[] body)
         {
             Method = method;
-            Header = header;
-            m_body0 = body;
-            m_bodyN = null;
+            Header = header;
+            if (body != null)

```

Comment:
Please use curly braces for single-statement conditionals. That appears to be the style in this codebase.